src/db_queries/expenses.ts

```diff
@@ -0,0 +1,120 @@
+import type CenterExpenses from "../models/center_expenses.model";
+import { Helpers } from '../utils/helpers';
+import { Request, Response } from 'express';
+import dbClient from "../db/client";
+import { QueryResult } from "pg";
+
+const helpers = new Helpers();
+
+export class Expenses {
+
+    private readExpenseQuery = "SELECT * FROM organizations_centers_expenses WHERE center_id = $1";
+    private createExpenseQuery = "INSERT INTO organizations_centers_expenses (center_id, month, year, expense, amount, is_recurring) VALUES ($1, $2, $3, $4, $5, $6)";
+    private deleteExpenseQuery = "DELETE FROM organizations_centers_expenses WHERE center_id = $1 AND id = $2";
+    private updateExpenseQuery = "UPDATE organizations_centers_expenses SET month = $1, year = $2, expense = $3, amount = $4, is_recurring = $5 WHERE center_id = $6 AND id = $7";
+    private totalExpenseQuery = "SELECT SUM(amount) FROM organizations_centers_expenses WHERE center_id = $1";
+    private totalExpensePaidQuery = "SELECT SUM(amount) FROM organizations_centers_expenses_paid WHERE center_id = $1";
+
+    constructor() {
+        helpers.autoBind(this);
+    }
+
+    private async getCenterExpenseQuery(centerId: string): Promise<QueryResult<CenterExpenses>> {
```

Comment:
Let's have this function name in plural as well (`getCenterExpensesQuery` instead of `getCenterExpenseQuery`).